content/docs/optimizing-performance.md

```diff
@@ -372,43 +370,44 @@ handleClick() {
 };
 ```
 
-You can also rewrite code that mutates objects to avoid mutation, in a similar way. For example, let's say we have an object named `colormap` and we want to write a function that changes `colormap.right` to be `'blue'`. We could write:
+También puedes reescribir el código que muta los objetos para evitar la mutación, de forma similar.
```

Comment:
Can you merge line 373 and 374 into one please? This reduce the risk of introducing any merge conflict in the future.

Reply:
Merged the lines, also fixed some extra line spaces (2) that were added somewhere along the way in the [last commit](https://github.com/reactjs/es.reactjs.org/pull/194/commits/9e6f3ae8df109dc044a989bf9f1b5df5b26a0247). Lines match the original document now.